FirebaseAuth/Tests/Unit/FIRAuthTests.m

```diff
@@ -433,6 +433,8 @@ - (void)testFetchSignInMethodsForEmailSuccess {
             });
           });
   XCTestExpectation *expectation = [self expectationWithDescription:@"callback"];
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Weverything"
```

Comment:
Could this be scoped down to the actual warning?

Reply:
Good point. Done.